update output.md and usage.md (#194)
nf-core/funcscan: docs/output.md

The output of nf-core/funcscan provides reports for each of the functional groups:

- antibiotic resistance genes (tools: [ABRicate](https://github.com/tseemann/abricate), [AMRFinderPlus](https://www.ncbi.nlm.nih.gov/pathogens/antimicrobial-resistance/AMRFinder), [DeepARG](https://bitbucket.org/gusphdproj/deeparg-ss/src/master), [fARGene](https://github.com/fannyhb/fargene), [RGI](https://card.mcmaster.ca/analyze/rgi) summarized by [hAMRonization](https://github.com/pha4ge/hAMRonization))
- antimicrobial peptides (tools: [macrel](https://github.com/BigDataBiology/macrel), [amplify](https://github.com/bcgsc/AMPlify), [ampir](https://ampir.marine-omics.net), [hmmsearch](http://hmmer.org) summarized by [AMPcombi](https://github.com/Darcy220606/AMPcombi))
- biosynthetic gene clusters (tools: [antiSMASH](https://docs.antismash.secondarymetabolites.org), [deepBGC](https://github.com/Merck/deepbgc), [GECCO](https://gecco.embl.de), [hmmsearch](http://hmmer.org) summarized by [comBGC](#combgc))

> Additional to summary reports ([ARGs](#hamronization), [AMPs](#ampcombi), [BGCs](#combgc)), the output directories from all applied tools are provided. This includes the functional annotation output from either [prokka](https://github.com/tseemann/prokka), [prodigal](https://github.com/hyattpd/Prodigal), or [Bakta](https://github.com/oschwengers/bakta) if the `--save_annotations` flag was set.

Reviewer: ```suggestion
As a general workflow, we recommend to first look at the to summary reports ([ARGs](#hamronization), [AMPs](#ampcombi), [BGCs](#combgc)), to get a general overview of what hits have been found across all the tools of each functional group. After which, you can explore the specific output directories  of each tool to get more detailed information about each results. The tool-specific output directories, also includes the functional annotation output from either [prokka](https://github.com/tseemann/prokka), [prodigal](https://github.com/hyattpd/Prodigal), or [Bakta](https://github.com/oschwengers/bakta) if the `--save_annotations` flag was set.
```

[AMPcombi](https://github.com/Darcy220606/AMPcombi) summarizes the results of **antimicrobial peptide (AMP)** prediction tools (AMPIR, AMPLIFY, MACREL, and other non nf-core tools) into a single table and aligns the hits against a reference AMP database for functional and taxonomic classification.

> 

Reviewer 2: ```suggestion
<details markdown="1">
<summary>AMP summary table headers</summary>

| Table column | Description |
|---|---|
| `name` | the name of the sample. |
| `contig_id` | the contig header. |
| `prob_macrel` | the probability associated with the AMP prediction using `MACREL`
| `prob_neubi` | the probability associated with the AMP prediction using `NEUBI`
| `prob_ampir` | the probability associated with the AMP prediction using `AMPIR`
| `prob_amplify` | the probability associated with the AMP prediction using `AMPLIFY`
| `evalue_hmmer` | the expected number of false positives (nonhomologous sequences) with a  similar of higher score. This stands for how significant te hit is, the lower the evalue, the
more significant the hit. |
| `aa_sequence` | the amino acid sequence that forms part of teh contig and is AMP encoding. |
| `target_id` | the [DRAMP](http://dramp.cpu-bioinfor.org/) ID within the database found to be similar to the predicted AMP by `DIAMOND` alignment. |
| `pident` | the percentage identity of amino acid residues that fully aligned between the `DRAMP` sequence and the predicted AMP sequence. |
| `evalue` | the number of alignments of similar or better qualities that can be expected when searching a database of similar size with a random sequence distribution. This is generated by `DIAMOND` alignments using the [DRAMP](http://dramp.cpu-bioinfor.org/) AMP database. The lower the value the more significant that the hit is positive. An e-value of < 0.001 means that the this hit will be found by chance once per 1,0000 queries. |
| `Sequence` | the sequence corresponding to the `DRAMP` ID found to be similar to the predicted AMP sequence. |
| `Sequence_length` | the number of amino acid residues in the `DRAMP` sequence. |
| `Name` | the full name of the peptide copied from teh database it was uploaded to. |
| `Swiss_Prot_Entry` | the entry name of the peptide within the [UniProtKB/Swiss-Prot](https://www.uniprot.org/help/entry_name) database. |
| `Family` | the name of teh family, group or class of AMPs this peptide belongs to. For e.g. bacteriocins. |
| `Gene` | the name of the gene (if available in the database) that encodes the peptide. |
| `Source` | the name of the source organism (if available in the database) from which the peptide was extracted from. |
| `Activity` | the activity of the peptide. For example, whether it is Antimicrobial, Antibacterial, Anti-Gram+, Anti-Gram-, Insecticidal or Antifungal. |
| `Protein_existence` | the status of the peptide. For example, whether it is only a homology, protein level, predicted or transcript level. |
| `Structure` | specify what type of structure the peptide has. For e.g. alpha helix, bridge, etc. |
| `Structure_Description` | specify any further description of the structure if available. |
| `PDB_ID` | the ID of an equivalent peptide found in the protein data bank [PDB](https://www.rcsb.org/docs/general-help/organization-of-3d-structures-in-the-protein-data-bank). |
| `Comments` | provide any further details found in the databse regarding the peptide. |
| `Target_Organism` | the name of the target organsim to which the peptide is effective against. |
| `Hemolytic_activity` | specify the type of hemolytic activity if any. |
| `Linear/Cyclic/Branched` | specify whether the hit is a linear, cyclic or branched peptide. |
| `N-terminal_Modification` | specify whether it contains N-terminal_Modification. |
| `C-termina_Modification` | specify whether it contains C-terminal_Modification. |
| `Other_Modifications` | specify whether there are any other modifications found in the peptide structure. |
| `Stereochemistry`— the type of peptide stereochemistry if available. |
| `Cytotoxicity` | the mechansim of cytotoxicity of the peptide if available. |
| `Binding_Target` | the binding target of the peptide. For e.g. lipid, cell membran or chitin binding. |
| `Pubmed_ID` | the Pubmed ID if a publication is associated with the peptide. |
| `Reference` | the citation of the associated publication if available. |
| `Author` | the author names associated with the publication or who have uploaded the peptide. |
| `Title` | the publication title if available. |

</details>
```

**comBGC** is a local module which summarizes the results of the **Biosynthetic Gene Cluster (BGC)** prediction tools (antiSMASH, deepBGC, GECCO) used in the pipeline into one comprehensive summary with standardized headers.

<details markdown="1">
<summary>BGC summary table headers</summary>

> - `Sample_ID`
> - `Prediction_tool`
> - `Contig_ID`
> - `Product_class`
> - `BGC_probability`
> - `BGC_complete`
> - `BGC_start`
> - `BGC_end`
> - `BGC_length`
> - `CDS_ID`
> - `CDS_count`
> - `PFAM_domains`
> - `MIBiG_ID`
> - `InterPro_ID`

Reviewer 3: ```suggestion
| Table column | Description |
|---|---|
| `Sample_ID` | IDs of your samples |
| `Prediction_tool` | BGC prediction tool (antiSMASH, DeepBGC, and/or GECCO) |
| `Contig_ID` | ID of the contig containing the candidate BGC |
| `Product_class` | Predicted BGC type/class |
| `BGC_probability`| Confidence of BGC candidate as inferred by the respective tool |
| `BGC_complete` | Whether BGC sequence is assumed to be complete or truncated by the edge of the contig |
| `BGC_start` | Predicted BGC start position on the ontig |
| `BGC_end` | Predicted BGC end position on the contig |
| `BGC_length` | Length of the predicted BGC |
| `CDS_ID` | ID of the coding sequence(s) (CDS) from the annotation step (prodigal/prokka/bakta) if provided by the tool |
| `CDS_count` | Number of CDSs the BGC contains |
| `PFAM_domains` | Inferred PFAM IDs or annotations if provided by the tool |
| `MIBiG_ID` | Inferred MIBiG IDs if provided by the tool |
| `InterPro_ID` | Inferred InterPro IDs if provided by the tool |
```
